common/tools/dev-tool/src/commands/run/extract-api.ts

```diff
@@ -117,12 +128,18 @@ export default leafCommand(commandInfo, async () => {
         dtsRollup: newDtsRollupOptions,
         docModel: newDocModel,
         apiReport: newApiReport,
-        mainEntryPointFilePath: exports[exprt].types,
+        mainEntryPointFilePath: exports[subpathExport].types,
       };
 
+      if (updatedConfigObject.messages?.extractorMessageReporting) {
+        updatedConfigObject.messages.extractorMessageReporting[
+          "ae-forgotten-export"
```

Comment:
TODO: comment why this is needed.

It's needed because otherwise you end up with a ton of warnings about forgotten exports if the subpath uses any of the `top level` exports. We'll want to figure that out, but for now it's probably better to avoid the noise it adds. Thoughts?